### PR TITLE
demo playbook and doc update

### PR DESCRIPTION
**Pull request by naveenkandakur**

### CODE REVIEW

File: playbooks/inventory [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [hmcs]
-9.3.147.114
+9.3.147.120
```
Comment (AnilVijayan) [inline playbooks/inventory]: No need to push this file

File: plugins/modules/powervm_lpar_instance.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
     - Anil Vijayan (@AnilVijayan)
 short_description: Create/Delete an AIX/Linux or IBMi partition
 notes:
-    - Currently supports creation of partition (powervm instance) with only processor and memory settings
+    - Currently supports creation of partition (powervm instance) with only processor and dedicated memory settings
```
Comment (AnilVijayan) [inline plugins/modules/powervm_lpar_instance.py]: should be like "Currently supports creation of partition (powervm instance) with only processor and memory settings on dedicated mode"

Comment (naveenkandakur) [reply plugins/modules/powervm_lpar_instance.py]: Done

File: docs/source/demo.rst [hunk 3]
```diff
@@ -48,3 +48,15 @@ policy then modify and activate it.
...
    :alt: 
+
+
+powervm_lpar_instance
+---------------------
+
+Create and Delete lpar instance
+"""""""""""""""""""""""""""""""
+
+The following gif demonstrates the Creating an aix lpar instance and deleting it.
```
Comment (AnilVijayan) [inline docs/source/demo.rst]: The following gif demonstrates the creation of an aix lpar instance and deletion of it.

**Review by AnilVijayan — CHANGES_REQUESTED** [review]
Please address the change requested

File: docs/source/demo.rst (hunk 3 above)
Comment (naveenkandakur) [reply docs/source/demo.rst]: Done

**Pull request merged**